charts/datasaur/templates/config/intelligence-env-configMap.yaml

```diff
@@ -2,8 +2,20 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: intelligence-env
-data: 
+data:
   NLTK_PROVIDER_CORENLP_URL: http://{{ .Release.Name }}-core-nlp-service:9000
-  NLTK_PROVIDER_PORT: '9090'
+  NLTK_PROVIDER_PORT: "9090"
   SPARKNLP_JSL_HOST: http://{{ .Release.Name }}-jsl-server-service:5000
-  SPARKNLP_PORT: '9091'
\ No newline at end of file
+  SPARKNLP_PORT: "9091"
+
+  # Datasaur Dinamic environment variables
+  DATASAUR_ASSIST_VULCAN_BASIC_AUTH: ZGF0YXNhdXI6ZGF0YXNhdXI=
```

Comment:
1. is this DATASAUR_ASSIST_VULCAN_BASIC_AUTH still relevant?
2. if yes, is this required?
3. how do we make sure that it is not hardcoded in helm chart as it may cause security issue since this value is accessible in the public repo

Reply:
![Screenshot from 2024-04-23 16-14-32](https://github.com/datasaur-ai/helm-charts/assets/100671309/29b72107-21c0-4a83-8396-6f0b4c0570e0)
apparently it is still required, used as bearer token.
screenshot is from datasaur-assist repo
hmmm, maybe we can blank the value, then tell whoever is gonna use the charts with datasaur-assist to fill in the code in values.yaml (we give the code directly), wdyt?

Reply:
sure i think it is better if the value is filled when the customer wants to set up.